examples/fineFoods/admin/mui/src/components/dashboard/orderTimeline/index.tsx

```diff
@@ -0,0 +1,152 @@
+//TODO: refine-mui doesn't have the following components:
+import {
+    Timeline,
+    TimelineConnector,
+    TimelineContent,
+    TimelineDot,
+    TimelineItem,
+    TimelineOppositeContent,
+    TimelineSeparator,
+} from "@mui/lab";
+import { useTranslate, useNavigation, useTable } from "@pankod/refine-core";
+import {
+    Box,
+    Button,
+    Tooltip,
+    Typography,
+    Pagination,
+} from "@pankod/refine-mui";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import { IOrder } from "interfaces";
+
+dayjs.extend(relativeTime);
+
+export const OrderTimeline: React.FC = () => {
+    const t = useTranslate();
+    const { show } = useNavigation();
+
+    const { tableQueryResult, current, setCurrent, pageCount } =
+        useTable<IOrder>({
+            resource: "orders",
+            initialSorter: [
+                {
+                    field: "createdAt",
+                    order: "desc",
+                },
+            ],
+            initialPageSize: 6,
+        });
+
+    const { data } = tableQueryResult;
+
+    const orderStatusColor = (
+        id: string,
+    ): { color: string; text: string; dotColor: string } => {
+        switch (id) {
+            case "1":
+                return {
+                    color: "#fff7e6",
+                    text: "pending",
+                    dotColor: "#ffa940",
+                };
+            case "2":
+                return { color: "#e6fffb", text: "ready", dotColor: "#36cfc9" };
+            case "3":
+                return {
+                    color: "#f6ffed",
+                    text: "on the way",
+                    dotColor: "#73d13d",
+                };
+            case "4":
+                return {
+                    color: "#e6f7ff",
+                    text: "delivered",
+                    dotColor: "#40a9ff",
+                };
+            default:
+                return {
+                    color: "#fff1f0",
+                    text: "cancelled",
+                    dotColor: "#ff4d4f",
+                };
+        }
+    };
+
+    return (
+        <>
+            <Timeline position="right">
+                {data?.data.map(({ createdAt, orderNumber, status, id }) => {
+                    const text = orderStatusColor(status.id.toString())?.text;
+                    const color = orderStatusColor(status.id.toString())?.color;
+                    const dotColor = orderStatusColor(
+                        status.id.toString(),
+                    )?.dotColor;
+
+                    return (
+                        <TimelineItem key={orderNumber}>
+                            <TimelineOppositeContent
+                                sx={{ display: "none" }}
+                            ></TimelineOppositeContent>
```

Comment:
```suggestion
                            <TimelineOppositeContent
                                sx={{ display: "none" }}
                           />
```

can it work like this